packages/console/src/components/CodeEditor/index.tsx

```diff
@@ -33,12 +33,20 @@ const CodeEditor = ({ language, isReadonly = false, value = '', onChange }: Prop
           onChange={handleChange}
         />
         <SyntaxHighlighter
+          wrapLongLines
+          codeTagProps={{
+            style: {
+              fontFamily: "'Roboto Mono', monospace",
+            },
+          }}
           customStyle={{
             background: 'transparent',
             fontSize: '14px',
             margin: '0',
             padding: '0',
             borderRadius: '0',
+            wordBreak: 'break-all',
+            fontFamily: "'Roboto Mono', monospace",
```

Comment:
why do we need to define same font family 3 times?

Reply:
The `pre` tag and its inner `code` tag both have default inline styles that come out of `react-syntax-highlighter`. I have to override the font-family for each tag element in order to make them work consistently